api/types/helmchartargs.go

```diff
@@ -63,6 +63,11 @@ type HelmChart struct {
 	// addition to either the default values file or the values specified in ValuesFile.
 	AdditionalValuesFiles []string `json:"additionalValuesFiles,omitempty" yaml:"additionalValuesFiles,omitempty"`
 
+	// AdditionalValuesFromFileContent holds value mappings where the value is
+	// specified in a file. This allows setting values from respective files.
+	// The file's content will become the value's content.
+	AdditionalValuesFromFileContent map[string]string `json:"additionalValuesFromFileContent,omitempty" yaml:"additionalValuesFromFileContent,omitempty"`
```

Comment:
I'm inclined to accept the feature request, but `additionalValuesFromFileContent` strikes me as verbose and unintuitive. Why not just call this field `setFile` or similar for parity with the flag?

Reply:
Changed the field to `setFile`